src/mono/mono/eventpipe/CMakeLists.txt

```diff
@@ -10,9 +10,9 @@ if(ENABLE_PERFTRACING)
         add_definitions(-DENABLE_PERFTRACING_PAL_TCP)
     endif (FEATURE_PERFTRACING_PAL_TCP)
 
-    if (FEATURE_PERFTRACING_DISABLE_LISTEN_PORTS)
```

Comment:
Why was this needed?

Reply:
It's a typo, the old cmake variable name was unused